models/index.js

```diff
@@ -0,0 +1,10 @@
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize(process.env.DB_URL)
+
+var models = [
+    'Todo',
+]
+models.forEach(function(model) {
+    module.exports[model] = sequelize.import(__dirname + '/' + model);
+});
```

Comment:
You shouldn't have a magic models import, you should just import models individually as you need them